### DISCUSSION TITLE
Overhaul Of UI And Other Updates

### DISCUSSION THREAD

**Discussion by Jonneal3** [mckaywrigley/chatbot-ui#1522] [Ideas]
Hey guys! I made a TON of custom mods into my chatbotUI and realized its come out a little "prettier" than the original. I also added a full api, auth for functions and a bunch of other things. Heres a video of it: 

https://www.loom.com/share/a64bddb4bee040ac8a2e87dbe115b61a?sid=32bf288a-3058-4901-8d37-a9e1985fadd8

Let me know your thoughts on what ppl have interest on adding vs. not adding. @mckaywrigley Would love you to weigh in as well!

Thanks! Jon

**Comment by secondtruth** [comment]
Great work, Jon! I think the connections feature would be a very useful addition.

**Comment by sebiweise** [comment]
Really nice work. Looks great 👍🏼

**Comment by csbenson001** [comment]
yes would love to see the connections capability!

**Comment by yashsinha-aero** [comment]
Thats amazing work! Perhaps more customization abilities like setting up a default model from a custom endpoint (like the custom models that u can currently add on chatbot ui), system prompts, temperature etc. Would be great

**Comment by carlosangelai** [comment]
The connections feature is amazing. Imagine connecting this feature, pardon the redundancy, to a unified API type service like Merge.dev It would make it possible to chat with an entire enterprise from one panel using multiple models. Could you also look into including full webhooks functionality? This would create an event stream of the activity on the platform that could be very useful for downstream or even upstream functions. Nice work!

**Comment by hadupa** [comment]
It looks like the video has been deleted. Is it no longer available anywhere? Very interested in this.
